whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/export/DatabaseTextExport.java

```diff
@@ -16,7 +16,7 @@ public DatabaseTextExport(final RpslObjectsExporter rpslObjectsExporter) {
     }
 
     @Override
-    @Scheduled(cron = "0 0 0 * * *")
+    @Scheduled(cron = "0 0 0 * * *", zone = EUROPE_AMSTERDAM)
```

Comment:
Perhaps comment that *this* job should run in Amsterdam time as the generated file on the FTP site will then appear at a consistent time

Reply:
done